plutus-core/executables/traceToStacks/TestGetStacks.hs

```diff
@@ -0,0 +1,78 @@
+import Common
+import Data.Text (Text, singleton)
+import Test.Tasty (defaultMain, testGroup)
+import Test.Tasty.HUnit (testCase, (@?=))
+
+toText :: Char -> Text
+toText = singleton
+
+xEvent :: [ProfileEvent]
+xEvent = [
+    MkProfileEvent 0 Enter (toText 'x'),
+    MkProfileEvent 1 Exit (toText 'x')
+    ]
+
+xStackVal :: [StackVal]
+xStackVal = [MkStackVal [toText 'x'] 1]
+
+zInyInxEvent :: [ProfileEvent]
```

Comment:
I think these could do with a sentence of haddock saying what kind of test case they're trying to set up.

Reply:
Like "simulates x calling y calling z"